packages/core/src/transports/types.ts

```diff
@@ -28,7 +28,7 @@ export interface TransportBody {
   exceptions?: ExceptionEvent[];
   logs?: LogEvent[];
   measurements?: MeasurementEvent[];
-  traces?: TraceEvent[];
+  traces?: TraceEvent;
```

Comment:
All the others are receiving an array of events in oppose to a single event. Shouldn't we maintain a consistency?

Reply:
OTEL traces payload already supports batching, so it's redudnant to have it at this level as well. Also receiver does not expect an array here. This is not a functional change, just a type fix